lib/hydra/works/services/determine_mime_type.rb

```diff
@@ -0,0 +1,24 @@
+module Hydra::Works
+  class DetermineMimeType
+    # Determines the mime type for a given file
+    # @param [IO, File, Rack::Multipart::UploadedFile, #read] file
+    # @param [String, NilClass] original_name of the file
+    # @return [String]
+    class << self
```

Comment:
FWIW, I would love to see some agreement on how we write our services and have some kind of automated enforcement that ensures we're consistent. Ruboservice anyone?